vignettes/diving_deeper.Rmd

```diff
@@ -80,7 +82,7 @@ heroes_information |>
   rmarkdown::paged_table()
 ```
 
-Now consider the `super_hero_powers` table, which contains data from a **repeating** instrument. In addition to `record_id` there is a `redcap_repeat_instance` column. The observational unit, or granularity, of each row is one **superpower per superhero**, identified by its `record_id` and `redcap_repeat_instance`. After these two **identifier columns** follow data columns and `form_status_complete`.
+Now consider the `super_hero_powers` table, which contains data from a **repeating** instrument. In addition to `record_id` there is a `redcap_repeat_instance` column. The observational unit, or granularity, of each row is one **superpower per superhero**, identified by its `record_id` and `redcap_repeat_instance`. Because these two columns uniquely identify a row in the data tibble, we call them **identifier columns**. In the REDCapTidieR supertibble, identifier columns always come first, followed by **data columns**. The last column is always `form_status_complete`. This column indicates whether the instrument has been marked off as completed.
```

Comment:
I don't think I saw anywhere the reference that the 734 non-repeat rows + the 5966 repeat rows adds up to the original 6700 from the original sparse block matrix. I think it's a nice little "aha!" to include.

Reply:
I decided to leave it out because this is actually more complicated. I think this might make people believe that the number of rows in the block matrix = sum(rows of each instrument) but that's not true. It's the sum of rows in all records plus the sum of rows in each nonrepeated instrument. Better leave this one alone.